imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -321,55 +328,120 @@ def test_partitioning_unstructured_with_well(
     simulation.write(original_dir, binary=False)
     simulation.run()
 
-    original_head = simulation.open_head()
-    original_flow_cbc = simulation.open_flow_budget()
+    expected_head = simulation.open_head()
+    expected_flow_cbc = simulation.open_flow_budget()
 
     # Partition the simulation, run it, and save the (merged) results
     split_simulation = simulation.split(partition_array)
 
     split_simulation.write(tmp_path, binary=False)
     split_simulation.run()
 
-    head = split_simulation.open_head()
-    head = head.ugrid.reindex_like(original_head)
+    actual_head = split_simulation.open_head()
+    actual_head = actual_head.ugrid.reindex_like(expected_head)
     # TODO:
     # Uncomment when fixed: https://gitlab.com/deltares/imod/imod-python/-/issues/683
```

Comment:
Boyscouting:
The issue mentioned in the comment is closed now, TODO can be removed

https://github.com/Deltares/imod-python/issues/683

Reply:
done, removed the TODO